sdk/identity/identity/test/internal/node/managedIdentityCredential.spec.ts

```diff
@@ -338,19 +335,16 @@ describe("ManagedIdentityCredential", function() {
     }
   });
 
-  // This fails on ubuntu1804_16x_node on Node version 16.3.0
-  it.skip("sends an authorization request correctly in an Azure Arc environment", async () => {
+  it("sends an authorization request correctly in an Azure Arc environment", async () => {
     // Trigger Azure Arc behavior by setting environment variables
 
     process.env.IMDS_ENDPOINT = "https://endpoint";
     process.env.IDENTITY_ENDPOINT = "https://endpoint";
 
-    const filePath = "path/to/file";
+    const date = Date.now();
+    const filePath = `test_${date}`;
```

Comment:
I have a couple concerns:

1. It's probably better to create the file in the system TEMP directory rather than the current directory.
2. If the test fails, the temp file might not be unlinked.  We could use a try/catch.

I wonder if the best way to address these is to use a package like `tmp` or `tmp-promise` which claims to handle both.

Reply:
I updated this use the tmp directory through os.tmpdir(), thank you!